photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/training/Driver.scala

```diff
@@ -402,7 +402,7 @@ object Driver {
         val logsDir = new Path(params.outputDir, LOGS).toString
         implicit val logger = new PhotonLogger(logsDir, sc)
         // TODO: This Photon log level should be made configurable
-        logger.setLogLevel(PhotonLogger.LogLevelInfo)
+        logger.setLogLevel(PhotonLogger.LogLevelDebug)
```

Comment:
I disagree - I'm not sure exactly which commit changed the logging level to "Info" but it has been "Debug" for a while before that, so I think we should keep it at "Debug" until the option becomes configurable.

My reasoning is that the log isn't too much larger in "Debug" mode, there have been no complaints, and it keeping it at "Info" would publish less info than users may be used to.